src/libcommonserver/db/sqlitedb.cpp

```diff
@@ -421,7 +421,13 @@ SyncName makeSyncName(sqlite3_value *value) {
 
 static void normalizeSyncName(sqlite3_context *context, int argc, sqlite3_value **argv) {
     if (argc == 1) {
-        SyncName normalizedName = Utility::normalizedSyncName(makeSyncName(argv[0]));
+        SyncName name(makeSyncName(argv[0]));
+        SyncName normalizedName;
+        if (!Utility::normalizedSyncName(name, normalizedName)) {
+            // TODO: Is there a better solution?
+            normalizedName = name;
+        }
```

Comment:
What happen if normalization fail? I do not have a better solution to offer anyway but just worried about the side effect

Reply:
Discussed together, ok

Reply:
In this case, the name won't be modified.
Ideally, the record should be deleted so that synchronization generates a pseudo-conflict.
Can be fixed in an other PR I think.